boards/arm/stm32/b-g474e-dpow1/configs/buckboost/defconfig

```diff
@@ -0,0 +1,126 @@
+#
+# This file is autogenerated: PLEASE DO NOT EDIT IT.
+#
+# You can use "make menuconfig" to make any modifications to the installed .config file.
+# You can then do "make savedefconfig" to generate a new defconfig file that includes your
+# modifications.
+#
+# CONFIG_ARCH_LEDS is not set
+CONFIG_ADC=y
+CONFIG_ANALOG=y
+CONFIG_ARCH="arm"
+CONFIG_ARCH_BOARD="b-g474e-dpow1"
+CONFIG_ARCH_BOARD_B_G474E_DPOW1=y
+CONFIG_ARCH_BUTTONS=y
+CONFIG_ARCH_CHIP="stm32"
+CONFIG_ARCH_CHIP_STM32=y
+CONFIG_ARCH_CHIP_STM32G474R=y
+CONFIG_ARCH_HIPRI_INTERRUPT=y
+CONFIG_ARCH_RAMVECTORS=y
+CONFIG_ARCH_STACKDUMP=y
+CONFIG_BOARD_LOOPSPERMSEC=16717
+CONFIG_BUILTIN=y
+CONFIG_DEBUG_FULLOPT=y
+CONFIG_DEBUG_SYMBOLS=y
+CONFIG_DISABLE_ENVIRON=y
+CONFIG_DISABLE_MQUEUE=y
+CONFIG_DISABLE_POSIX_TIMERS=y
+CONFIG_DISABLE_PTHREAD=y
+CONFIG_DRIVERS_SMPS=y
+CONFIG_EXAMPLES_SMPS=y
+CONFIG_EXAMPLES_SMPS_DEVPATH="/dev/smps0"
+CONFIG_EXAMPLES_SMPS_IN_VOLTAGE_LIMIT=10000
+CONFIG_EXAMPLES_SMPS_OUT_CURRENT_LIMIT=100
+CONFIG_EXAMPLES_SMPS_OUT_POWER_LIMIT=100
+CONFIG_EXAMPLES_SMPS_OUT_VOLTAGE_DEFAULT=5000
+CONFIG_EXAMPLES_SMPS_OUT_VOLTAGE_LIMIT=10000
+CONFIG_EXAMPLES_SMPS_TIME_DEFAULT=10
+CONFIG_FDCLONE_STDIO=y
+CONFIG_INIT_ENTRYPOINT="nsh_main"
+CONFIG_INIT_STACKSIZE=1024
+CONFIG_INTELHEX_BINARY=y
+CONFIG_LIBDSP=y
+CONFIG_LIBM=y
+CONFIG_NAME_MAX=16
+CONFIG_NSH_ARCHINIT=y
+CONFIG_NSH_BUILTIN_APPS=y
+CONFIG_NSH_DISABLE_BASENAME=y
```

Comment:
STM32G474 has 512 Kbytes of Flash so we don't have to spare memory like it is for stm32f334. We can remove this all `CONFIG_NSH_DISABLE` options